tools/SqliteCompiler/Program.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SqliteCompiler
+{
+    public class Program
```

Comment:
It feels to me that this should be just a script - it is probably a few lines of shell script (or maybe even one if you put a few commands on one line)